tests/test.py

```diff
@@ -1036,6 +1041,37 @@ def fork_connector(self, connector_port: int, acraserver_port: int,
         logging.info("fork connector finished [pid={}]".format(process.pid))
         return process
 
+    def fork_ocsp_server(self, port: int, check_connection: bool=True):
+        logging.info("fork OpenSSL OCSP server with port {}".format(port))
+
+        ocsp_server_connection = self.get_ocsp_server_connection_string(port)
+
+        args = {
+            'port': port,
+            'index': TEST_TLS_OCSP_INDEX,
+            'rsigner': TEST_TLS_OCSP_CERT,
+            'rkey': TEST_TLS_OCSP_KEY,
+            'CA': TEST_TLS_CA,
+        }
+
+        cli_args = sorted(['-{}={}'.format(k, v) for k, v in args.items()])
+        print('openssl ocsp args: {}'.format(' '.join(cli_args)))
+
+        process = self.fork(lambda: subprocess.Popen(['openssl', 'ocsp'] + cli_args))
+
+        # Openssl OCSP server exitx on first invalid request, let's skip connection check for now
```

Comment:
oh crap )) but I predict that when acra-server/connector will check ocsp URI's by test query to OCSP server, our tests will start blinking, because sometimes OCSP server may up longer and acra will check before it up.
So better to make some valid request here by some python library or create some template of request as byte array and hardcode it here, and make correct HTTP request with payload from template.

Reply:
> sometimes OCSP server may up longer and acra will check before it up

It shouldn't be slow actually. But I agree, we may need more stable OCSP server instead of openssl.

Reply:
we started call `check_connection` after each fork not just for fun) it was a frequent case on slow machines (circleci for example) when acra hadn't started yet when testcase tried to connect.